src/camp/generators/create_sql.py

```diff
@@ -345,11 +345,11 @@ def val_or_none(obj, attr='value'):
                 return ''
             return getattr(obj, attr)
 
-        name = val_or_none(self.admset.get_child(self.adm, models.Mdat, 'name'))
-        ns = val_or_none(self.admset.get_child(self.adm, models.Mdat, 'namespace'))
-        version = val_or_none(self.admset.get_child(self.adm, models.Mdat, 'version'))
-        org = val_or_none(self.admset.get_child(self.adm, models.Mdat, 'organization'))
-        desc = escape_description_sql(val_or_none(self.admset.get_child(self.adm, models.Mdat, 'namespace'), 'description'))
+        name = val_or_none(self.admset.get_child(self.adm, models.Ident, 'name'))
```

Comment:
The notion of "Mdat" (metadata) is replaced by alternatives:
* name is direct `self.adm.name`
* ns is no longer a separate concept; the ADM name is the text form of namespace
* Others are named from keys of [`MOD_META_KYWDS`](https://github.com/JHUAPL-DTNMA/dtnma-ace/blob/57fa2e540e3c9d181db6b98ad496688449b03268/src/ace/adm_yang.py#L76)